app/models/plan.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Plan < ApplicationRecord
+    has_many :Features
```

Comment:
why is it capital, should be :features. where are dependent options.

Reply:
updated